code401Challenges/src/main/java/code401Challenges/ArrayShift.java

```diff
@@ -0,0 +1,31 @@
+package code401Challenges;
+
+public class ArrayShift {
+
```

Comment:
You're correct! the main difference between libraries and applications is that libraries don't have a main method.